savant/run.py

```diff
@@ -1,7 +1,7 @@
 from flask import Flask
 import os
 
-from app import app
+from app import *
```

Comment:
Please don't use `import *`. You need to call out each import individually. For instance `from app import app,thing`